lib/matplotlib/transforms.py

```diff
@@ -36,7 +36,7 @@
 import functools
 import textwrap
 import weakref
-
+import math
```

Comment:
This belongs with stdlib imports above.

Reply:
You keep marking this resolved, but it isn't.

Reply:
weakref is also a stdlib import, so is textwrap?

Reply:
Not sure if that's the point here, but the import groups should each be in lexical order.

Reply:
This replaces the blank line, so it's merging the first and second blocks.

Reply:
Ok, now I get it. The first comment was very unclear to me.